### ISSUE TITLE
Missing cool down option for run trained with constant learning rate

### ISSUE THREAD

**Issue by clessig** [ecmwf/WeatherGenerator#1199]
We recently switched to constant learning rate. Validation results are signficantly besser when this is combined with a "short" cool down run that anneals the learning rate to 0. We currently do not have a simple way to run from a given epoch / the last checkpoint this annealing (it can be realized with adjusting num_epochs but this is imprecise and error prone).
